solr/core/src/test/org/apache/solr/util/stats/MetricUtilsTest.java

```diff
@@ -103,16 +108,29 @@ public void testMetrics() throws Exception {
       } else if (k.startsWith("histogram")) {
         assertEquals(1L, v.get("count"));
       } else if (k.startsWith("aggregate")) {
-        assertEquals(2, v.get("count"));
+        assertEquals(4, v.get("count"));
         Map<String, Object> values = (Map<String, Object>)v.get("values");
         assertNotNull(values);
-        assertEquals(2, values.size());
+        assertEquals(4, values.size());
         Map<String, Object> update = (Map<String, Object>)values.get("foo");
         assertEquals(10, update.get("value"));
         assertEquals(1, update.get("updateCount"));
         update = (Map<String, Object>)values.get("bar");
         assertEquals(2, update.get("value"));
+        // assertEquals(1.5, update.get("mean"));
```

Comment:
This and the other temporarily commented-out tests report `null` instead of the expected value. I'll dig into this further, but if anyone can spot why this fails please let me know!

Reply:
I figured it out!